services/httpd/handler.go

```diff
@@ -108,6 +110,11 @@ type Handler struct {
 		WritePoints(database, retentionPolicy string, consistencyLevel models.ConsistencyLevel, user meta.User, points []models.Point) error
 	}
 
+	Store interface {
```

Comment:
Do you have a strong preference for inline interface definitions vs promoting it to a package-level type? i.e.

```go
type Store interface {
    Read(ctx context.Context, req *storage.ReadRequest) (storage.Results, error)
    WithLogger(log *zap.Logger)
}
```

My case is that some tools appear to have a harder time identifying inlined interfaces and subsequently other types which implement this interface.